wgsl_bindgen/src/lib.rs

```diff
@@ -67,6 +67,7 @@ mod wgsl_type;
 pub use types::*;
 pub use wgsl_bindgen::*;
 pub use wgsl_type::*;
+pub use naga::valid::Capabilities;
```

Comment:
Before re-exporting just as is, Can we re-alias this as `WgslShaderIRCapabilities` and export as such.

Reply:
Actually would also instead remove this line and export from `wgsl_bindgen.rs` file instead. Since it's pretty much just exporting everything from there

Reply:
Renamed and moved :)